support/client/lib/vwf/model/jPlayer.js

```diff
@@ -361,12 +362,21 @@ define( [
 
                 switch( methodName ) {
                     
-                    case "play":
+                    case "load":
+                        if( node.url ) {
+                            node.jPlayerElement.jPlayer( "load" ); 
+                            this.logger.infox( "Loading!" ); 
+                        } else {
+                            this.logger.errorx( "No URL given!" ); 
+                        }
+                        break;
 
-                        if( methodParameters ){
-                            this.kernel.setProperty( node.ID, "url", methodParameters );
+                    case "play":
+                        if( node.url ) {
+                            node.jPlayerElement.jPlayer( "play" ); 
+                        } else {
+                            this.logger.errorx( "No URL given!" ); 
```

Comment:
The load command does not set the URL. I'm following the convention that JPlayer uses - you set the URL first (With our driver, it's a set of the url property. With JPlayer, it's in the [setMedia command](http://jplayer.org/latest/developer-guide/#jPlayer-setMedia)), and the [load method](http://jplayer.org/latest/developer-guide/#jPlayer-load) tells it to preload what was at the previously-set URL.